MonoZelda/MonoZeldaGame.cs

```diff
@@ -78,6 +80,11 @@ protected override void LoadContent()
 
     protected override void Update(GameTime gameTime)
     {
+        if(PlayerState.ObtainedTriforce == true)
+        {
+            this.Exit();
+        }
```

Comment:
Some sort of event or command to trigger this would be better than checking player state every single game update